.github/workflows/ci.yaml

```diff
@@ -29,3 +29,9 @@ jobs:
           {"folder": ".", "bazelversion": "6.4.0"},
           {"folder": ".", "bzlmodEnabled": true}
         ]
+
+  verify-bcr-patches:
```

Comment:
oops, this should have been in the prior PR where I added a patch file - we want to check when it needs to be recreated.